src/lib/ContextMenus.tsx

```diff
@@ -140,6 +141,7 @@ function ContextMenus(props: Props) {
     const status = useContext(StatusContext);
     const isOnline = status === ClientStatus.ONLINE;
     const history = useHistory();
+    const theme = useContext(ThemeContext);
```

Comment:
Instead of loading the theme and using `theme.error`, using `var(--error)` as the color should be preferred.